README.md

```diff
@@ -22,7 +22,4 @@ Tutorial will be helpful to you for customizing template-base!)
 
```

Comment:
@daisukenishino 

Please divide this file into English version and Japanese version.